eng/pipelines/corefx-base.yml

```diff
@@ -189,6 +189,8 @@ jobs:
 
             - template: /eng/pipelines/helix.yml
               parameters:
+                # send tests to helix only on public builds, official scheduled builds or manual official builds.
+                condition: or(eq(parameters.isOfficialBuild, 'false'), and(notIn(variables['Build.Reason'], 'BatchedCI'), notIn(variables['Build.Reason'], 'IndividualCI')))
```

Comment:
FYI, variable templates are a thing now.  Not sure if that is something you want to use in your templates or not - https://github.com/Microsoft/azure-pipelines-yaml/blob/master/design/variables.md

Reply:
Hmm interesting, thanks for the pointer 😄